fireant/slicer/filters.py

```diff
@@ -61,10 +61,10 @@ def __init__(self, dimension_definition, start, stop):
 
 class PatternFilter(DimensionFilter):
     def _apply(self, dimension_definition, patterns):
-        definition = dimension_definition.like(patterns[0])
```

Comment:
FYI there is no such thing as ilike in mysql, so this will not work there.

Reply:
Replaced it with lower